agent_deploy/kubernetes/sysdig-image-analyzer-daemonset-v1.yaml

```diff
@@ -0,0 +1,142 @@
+# apiVersion: extensions/v1beta1  # If you are in Kubernetes version 1.8 or less please use this line instead of the following one
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sysdig-image-analyzer
+  labels:
+    app: sysdig-image-analyzer
+spec:
+  selector:
+    matchLabels:
+      app: sysdig-image-analyzer
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: sysdig-image-analyzer
+    spec:
+      volumes:
+      # Needed for cri-o image inspection.
+      # cri-o and especially OCP 4.x by default use containers/storage to handle images, and this makes sure that the
+      # analyzer has access to the configuration. This file is mounted read-only.
+      - name: etccontainersstorage-vol
```

Comment:
just curious about naming. Why smash this name together without dashes?

Reply:
Because we looked at the agent daemonset that has things like `varrun-vol` and got a little carried away. We can add dashes there if you think that'd make it more readable.

Reply:
yeah, this seems unnecessary and unreadable

Reply:
you got it